installer/x86_64/install.sh

```diff
@@ -489,11 +489,14 @@ set timeout=5
 
 EOF
 
-# Add the logic to support grub-reboot
+# Add the logic to support grub-reboot and grub-set-default
 cat <<EOF >> $grub_cfg
 if [ -s \$prefix/grubenv ]; then
   load_env
 fi
+if [ "\${saved_entry}" ] ; then
```

Comment:
Its needed for https://github.com/Azure/sonic-utilities/pull/40 for user to set default image for boot.

grub-set-default utility writes to value of saved_entry variable to grubenv.
https://www.gnu.org/software/grub/manual/legacy/Invoking-grub_002dset_002ddefault.html
This patch provides support for grub-set-default to allow user choose a default image to boot from.